packages/plugins/i18n/server/services/entity-service-decorator.js

```diff
@@ -167,18 +166,14 @@ const decorator = (service) => ({
 
     const { kind } = strapi.getModel(uid);
 
-    const wrappedParams = await this.wrapParams(opts, { uid, action: 'findMany' });
-
-    const query = transformParamsToQuery(uid, wrappedParams);
-
     if (kind === 'singleType') {
       if (opts[LOCALE_QUERY_FILTER] === 'all') {
-        return strapi.db.query(uid).findMany(query);
+        return service.findMany.call(this, uid, opts, { ...opts, ignoreKind: true });
```

Comment:
I think this should be:
```suggestion
        return service.findMany.call(this, uid, { ...opts, ignoreKind: true });
```